hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -48,6 +48,19 @@
       name:    "cs_aes_halt"
       act:     "req"
       package: "entropy_src_pkg"
+      desc:    '''
+               Coordinate activity between CSRNG's AES and Entropy Source's SHA3. The idea is that
+               Entropy Source requests CSRNG's AES to halt and waits for CSRNG to acknowledge before
+               it starts its SHA3. While SHA3 runs, Entropy Source keeps the request high. CSRNG may
+               not drop the acknowledge before Entropy Source drops the request.
+
+               Current limitations:
+               1. During startup and in Firmware Override - Extract & Insert mode, Entropy Source
+                  makes no AES Halt requests but still activates its SHA3 engine.
+               2. Outside Firmware Override - Extract & Insert mode, Entropy Source may occasionally
+                  activate its SHA3 engine without requesting AES Halt, but no more than for one
+                  Keccak round (25 clock cycles) every 512 clock cycles.
```

Comment:
I think this needs some clarification. IIRC a single unmasked Keccak round takes exactly 1 clock cycle and a full Keccak takes 24 rounds i.e., 24 clock cycles. The 25 probably comes from loading new entropy into the state (1600 bits in 64-bit chunks == 25 chunks). The loading itself isn't critical as per chunk it's just XORing 64-bits and updating 64 FFs. At the very end, the Keccak round can be executed which will use a lot more power (lots of logic + updating 1600 FFs) - this is what we care about.

Reply:
Right, let me clarify: When `keccak_round` gets triggered, it always does 24 rounds, which take 24 clock cycles (since it's unmasked). I'll update the description to reflect that.

(The additional 25th cycle was probably a surplus for loading data or control. When directly observing `u_sha3.u_keccak.keccak_st`, the activity period is reduced to 24 cycles.)

> Because it might mean the interface really doesn't prevent regular concurrent operation of both CSRNG's AES and ENTROPY_SRC's SHA3.

Correct. I'll remove the "occasionally" from the phrasing of limitation 2, because during continuous HT, Entropy Source runs its Keccak mostly without `cs_aes_halt` handshakes. How often depends on the window size; for a size of 1024, Keccak runs 4 times without handshake and once with handshake (for a size of 512, twice without and once with; and so on). I'm updating https://github.com/lowRISC/opentitan/issues/17941 to include these findings. Fixing this is out of scope for M2.5, but we should definitely revisit this for a later milestone (as part of #17941's resolution).